src/modify.js

```diff
@@ -12,14 +14,46 @@ function shortToFullPath(path) {
 }
 
 function standardizeAttrs(attrs) {
-  const { errorMessage, properties, ...rest } = attrs;
+  const {
+    errorMessage, // to be renamed
+    properties, // ignored because of recurisve call
```

Comment:
```suggestion
    properties, // ignored because of recursive call
```

Reply:
Oh god, it's official. I don't know how to type recurisive.